scripts/jenkins/ci1/freshvm

```diff
@@ -20,16 +20,13 @@ if ! [ -f $p/$flavor.img -o -f $p/$flavor.qcow2 ]; then
 fi
 
 virsh destroy $n
-if [ "$n" = "cleanvm" ] ; then
-    dd if=/dev/zero of=/dev/vg0/$n-data count=16
-    if [ -e $p/$flavor.qcow2 ] ; then
-        time qemu-img convert $p/$flavor.qcow2 /dev/vg0/$n
-    else
-        time dd if=$p/$flavor.img of=/dev/vg0/$n bs=64k
-    fi
+dd if=/dev/zero of=/dev/vg0/$n-data count=16
```

Comment:
> and not neccessary all VMs deployed this way have a *-data volume, so I recommend to add:
> if [[ -e /dev/vg0/$n-data ]] ; then ...

Done in https://github.com/toabctl/automation/commit/793830145eb2111517d4299e8036db8dca73ee19